src/pages/RegisterPage/index.js

```diff
@@ -15,169 +15,158 @@ const RegisterPagePage = () => {
+          <div
+            className={`${registerPageStyles["input-box-set-master"]} border-1px-mischka`}
           >
-            <h1
-              className={`${registerPageStyles["text-4"]} nunitosans-bold-white-26px`}
-            >
-              {`Sign up`}
-            </h1>
-          </button>
-
-          <div className={registerPageStyles["login-referral"]}>
-            <div
-              className={`${registerPageStyles["have-an-account"]} nunitosans-normal-black-28px`}
-            >
-              {`Have an account?`}
-            </div>
-            <div
-              className={`${registerPageStyles["text-5"]} nunitosans-bold-licorice-28px`}
-              onClick={handleNavigate16}
-            >
-              {`Log in now`}
+            <div className={registerPageStyles["input-text"]}>
+              <div
+                className={`${registerPageStyles["text"]} nunitosans-normal-storm-gray-28px`}
+              >
+                <Input
+                  type="password"
+                  value={confirmPassword}
+                  onChange={(e) => setConfirmPassword(e.target.value)}
+                  placeholder="Enter confirmed password"
+                ></Input>
+              </div>
             </div>
           </div>
         </div>
-     
 
-      
-        <div
-          style={{ backgroundImage: `url(${"/images/img_image5.png"})` }}
-          className={registerPageStyles["side-panel"]}
+        <button
+          className={registerPageStyles["button-master"]}
+          onClick={(e) => {
+            e.preventDefault();
+            handleSignUp(email, password, username, country, navigate);
+          }}
         >
-          <img
-            src="/images/img_image2.png"
-            className={registerPageStyles["image-2"]}
-            alt="image2"
-          />
-          <img
-            src="/images/img_image3.png"
-            className={registerPageStyles["image-3"]}
-            alt="image3"
-          />
-          <img
-            src="/images/img_image4.png"
-            className={registerPageStyles["image-4"]}
-            alt="image4"
-          />
-
           <h1
-            align="left"
-            className={`${registerPageStyles["anytime-anywhere-with-minerva"]} merriweather-bold-black-60px`}
+            className={`${registerPageStyles["text-4"]} nunitosans-bold-white-26px`}
           >
-            Anytime
-            <br />
-            <br />
-            Anywhere
-            <br />
-            <br />
-            with Minerva
+            {`Sign up`}
           </h1>
+        </button>
+
+        <div className={registerPageStyles["login-referral"]}>
+          <div
+            className={`${registerPageStyles["have-an-account"]} nunitosans-normal-black-28px`}
+          >
+            {`Have an account?`}
+          </div>
+          <div
+            className={`${registerPageStyles["text-5"]} nunitosans-bold-licorice-28px`}
+            onClick={handleNavigate16}
```

Comment:
Same suggestion as with `handleNavigate21` -- consider using `navigate("/loginpage")` directly or changing its name to something like `navigateLogin`